### PR TITLE
Bump ZigBee libraries to 1.1.0

### PR DESCRIPTION
**Pull request by cdjackson**
Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
     <groupId>org.openhab.deps</groupId>
     <artifactId>pom</artifactId>
-    <version>1.0.30</version>
+    <version>1.0.31</version>
```
Comment (kaikreuzer) [inline pom.xml]: You have to change the project version throughout the repo, see our past updates.

**Comment by cdjackson** [comment]
Sorry - I can’t find a previous conversation about this, but I see what you mean. Is there a maven goal to perform this or is it done manually.

**Comment by kaikreuzer** [comment]
Here's some help on how to do it all at once: https://github.com/openhab/openhab-deps-repo/blob/master/pom.xml#L224

**Comment by cdjackson** [comment]
Thanks.

**Comment by cdjackson** [comment]
Thanks Kai.

**Pull request merged**